crux-http-server/src/crux/http_server.clj

```diff
@@ -434,23 +463,34 @@
                                                                (instant/read-instant-date))})
           entity-map (api/entity db eid)]
       {:status (if (some? entity-map) 200 404)
-       :body (when entity-map
-               (if (= (get-in request [:muuntaja/response :format]) "text/html")
-                 (let [linked-entities (link-all-entities db  "/_entity" entity-map)]
-                   (html5 (entity->html linked-entities entity-map)))
-                 entity-map))})))
+       :body (cond
+               (= (get-in request [:muuntaja/response :format]) "text/html") (if entity-map
+                                                                               (let [linked-entities (link-all-entities db  "/_entity" entity-map)]
+                                                                                 (raw-html
+                                                                                  {:body (entity->html linked-entities entity-map)
+                                                                                   :title "/_entity"
+                                                                                   :options options}))
+                                                                               (raw-html {:body [:div "No entity found"]
+                                                                                          :title "/_entity"
+                                                                                          :options options}))
+               (get query-params "link-entities?") {"linked-entities" (link-all-entities db  "/_entity" entity-map)
```

Comment:
shame that we have to have a different response format here if the user's requested linked entities - any way we can include these inline, say?